Dockerfile

```diff
@@ -2,17 +2,18 @@ FROM node
 
 MAINTAINER Michael R. Bernstein
 
-RUN useradd -u 9000 -r -s /bin/false app
+WORKDIR /usr/src/app/
+
+COPY engine.json /usr/src/app/
```

Comment:
@GordonDiggs Got it. thanks!

Sorry misread and thought you were referring to the trailing slashes after `app`